src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -1030,7 +1030,7 @@ namespace HVACVariableRefrigerantFlow {
                             VRF(VRFCond).HREIRFTHeatConst = CurveValue(HREIRFT, VRF(VRFCond).VRFCondPLR);
                         }
                     }
-                    HREIRFTConst = VRF(VRFCond).HRCAPFTHeatConst;
+                    HREIRFTConst = VRF(VRFCond).HREIRFTHeatConst;
```

Comment:
Actual fix. I did a few greps to ensure this was the only case where *eir* was assigned *cap* or vice versa